packages/frontend-acceptance-tests/src/step-definitions/Easy-renewal-login/I_am_at_the_start_of_the_renewal_journey.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+const { defineStep } = require('cucumber')
+
+const { createPermission } = require('../../lib/createPermissions')
+const RenewalsPage = require('../../pages/renew-login')
+
+defineStep(/^I am at the start of the renewal journey with a licence that expired yesterday$/, function () {
+  browser.call(async () => {
+    const permission = await createPermission()
+    const referenceNumber = permission.licence.referenceNumber
+    const lastSixDigits = referenceNumber.substr(referenceNumber.length - 6)
+
+    await browser.url('/') // TODO remove once IWTF-1877 has been completed
```

Comment:
Please can you put this in the JIRA ticket, rather than the code? 'TODO' comments have a nasty habit of hanging around